containers/blockchain/shopBackend/app.js

```diff
@@ -60,6 +62,22 @@ const utils = require('./utils/util');
       utils.queueRequest(params, executeEvent);
     });
   });
+
+  // pass params to iot dashboard
+  function iotDashboard(data) {
+    var options = {
+      method: 'GET',
+      uri: 'https://secretmap.mybluemix.net/steps?message='+JSON.stringify(data)
```

Comment:
We're constantly testing the blockchain network (enrolling users, creating transactions, etc..). Your app may receive some data from our tests when we roll this out. Just a heads up.
_I could also just comment out line 47 before building this image when I do some tests so that I don't flood you with block contents._

Reply:
I'm parsing whatever comes in and will pick up 2 or 3 metrics for display. Let's keep it uncommented for I need to test and proceed further on IoT open issues.